bottomsheet-commons/src/main/java/com/flipboard/bottomsheet/commons/ImagePickerSheetView.java

```diff
@@ -191,7 +192,13 @@ protected ImagePickerSheetView(final Builder builder) {
 
         // Set up the grid
         tileGrid = (GridView) findViewById(R.id.grid);
-        spacing = getResources().getDimensionPixelSize(R.dimen.bottomsheet_image_tile_spacing);
+
+        if(builder.spacing>0){
```

Comment:
`0` can be valid spacing. This should be `>= 0`, and `spacing` should be initialized as `-1`